### PR TITLE
itemci.com #24520

### PR DESCRIPTION
**Pull request by slavaleleka**
.

### CODE REVIEW

**Review by BlazDT — CHANGES_REQUESTED** [review]
Move the rules to Turkish filter.

**Comment by Alex-302** [comment]
@BlazDT

File: TurkishFilter/sections/specific.txt [hunk 1]
```diff
@@ -95,6 +95,10 @@ ceptenmp3.net##a[href^="https://www.onclickmega.com/"]
 konhaber.com###acilisreklam
 ||konhaber.com/bot/doviz/index.php
 diziay.com###leosbox
+||itemci.com/jsrc/jquery.fancybox.min.js$domain=itemci.com
```
[the remark this answers is not in the document]
Comment (Alex-302) [reply TurkishFilter/sections/specific.txt]: also this rule breaks the slider at the top

[the remark this answers is not in the document]
Comment (Alex-302) [reply TurkishFilter/sections/specific.txt]: better use extended CSS
```
itemci.com#$#html.fancybox-lock:has(> body) { overflow: visible!important; }
itemci.com#$#html.fancybox-lock:has(> body) > body { overflow: visible!important; }
itemci.com#$#body > div.fancybox-overlay:has(> div) { display: none!important; }
```

**Review by Alex-302 — CHANGES_REQUESTED** [review]

**Review by Alex-302 — CHANGES_REQUESTED** [review]

**Review by Alex-302 — APPROVED** [review]

**Pull request merged**